recipes/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+git clone --recursive https://github.com/ekg/glia.git && \
```

Comment:
@snewhouse you have commited here two recipes (the same). This one should be removed as well as the recipes/meta.yml